src/Symfony/Component/String/Tests/ByteStringTest.php

```diff
@@ -12,6 +12,7 @@
 namespace Symfony\Component\String\Tests;
 
 use Symfony\Component\String\AbstractString;
+use function Symfony\Component\String\b;
```

Comment:
not sure about the order, it's strange to have it in the middle

Reply:
I suppose Symfony style doesn't have a rule about functions, here, the alphabet order is used.

I think the Symfony coding style can be updated to have function separated (before or after other types )?

but that's out of the scope of the PR :) this is simply following Symfony coding style.